internal/cmd/skupper/connector/nonkube/connector_create.go

```diff
@@ -112,7 +112,10 @@ func (cmd *CmdConnectorCreate) ValidateInput(args []string) []error {
 		validationErrors = append(validationErrors, fmt.Errorf("host name must be configured: an IP address or hostname is expected"))
 	}
 	if cmd.Flags.TlsCredentials != "" {
-		// TBD what is valid TlsCredentials
+		ok, err := resourceStringValidator.Evaluate(cmd.Flags.TlsCredentials)
+		if !ok {
+			validationErrors = append(validationErrors, fmt.Errorf("tlsCredentials is not valid: %s", err))
```

Comment:
`tlsCredentials value is not valid` ?

Reply:
for now Fernando said to use resource validator.   I'll remove the comment

Reply:
I meant that maybe we could add the word `value` to the error message. If it doesn't make sense ignore this comment